watertap/data/techno_economic/injection_well_disposal.yaml

```diff
@@ -2,29 +2,21 @@ default:
   energy_electric_flow_vol_inlet:
     value: 0.409594654
     units: kWh/m^3
-  specific_capital_cost_flow_vol_inlet:
-    flow_basis_par:
+  capital_cost:
+    basis: flow_vol
+    cost_factor: None
+    reference_state:
       value: 473.176473
       units: m^3/hr
-    cap_basis_par:
+    capital_a_parameter:
       value: 22.5
-      units: M$
-    cap_exp_par:
+      units: MUSD_2011
+    capital_b_parameter:
       value: 0.7
       units: dimensionless
-    vol units:
-      value: m^3/hr
-    capital cost units:
-      value: M$
-    year:
-      value: 2011
-    cost_function_form:
-      function f(flow_vol_in): cap_basis_par*(flow_vol_in/flow_basis_par)^cap_exp
-      units: $
   recovery_frac_mass_H2O:
```

Comment:
Why does this pass-through unit has water recovery and removal? Just a little curious here because I delete these two attributes when I create a pass-through model and I should add the unit to the pass-through list in the `test_unit_parameter_files.py`. And I don't see any test with respect to the recovery and removal.

Reply:
Great point, I neglected to remove them when updating the .yaml. I will remove them and ensure tests are passing.